templates/nginx/repo.conf.erb

```diff
@@ -1,8 +1,20 @@
 server {
 	listen 80 default_server;
-	server_name repo;
+	<% if @letsencrypt_enabled %>
+	listen 443 ssl default_server;
```

Comment:
`default_server` here is a per-address and per-port setting so these two do not conflict with each other.

The directive makes it so that even if someone is just requesting the server by IP address this site will be served for HTTP (80) or HTTPS (443) even without knowing the requested server name.